dali/operators/generic/slice/slice_attr.cc

```diff
@@ -19,6 +19,31 @@
 
 namespace dali {
 
+void ProcessAxesArgs(std::vector<int> &axes, TensorLayout &axis_names, const OpSpec &spec,
+                     const char* axes_arg_name, const char* axis_names_arg_name) {
+  axes.clear();
+  axis_names = TensorLayout{};
+  const bool use_axes = axes_arg_name != nullptr;
+  const bool use_axis_names = axis_names_arg_name != nullptr;
```

Comment:
These names are a bit misleading - it doesn't really mean we will indeed use axis_names but if we even _can_ use them. I'd find it more readable to just use the name pointers in the condition.